extensions/2.0/Khronos/KHR_variant_materials/schema/mesh.primitive.FB_material_variants.schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
```

Comment:
The filename for this should be KHR_variant_materials.schema.json no?

Reply:
good catch, will fix with latest rename